packages/foreman/rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -114,6 +115,9 @@ install -D -m0644 %{SOURCE1} %{buildroot}%{_sysconfdir}/logrotate.d/%{gem_name}
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Oct 24 2024 Evgeni Golov - 1:1.8.1-2
+- Require satellite-maintain >= 0.0.3 if present
```

Comment:
Looks like this doesn't 100% cover it anymore

Reply:
well, damn you history rewriting in git :D